x/slashing/simulation/operations_test.go

```diff
@@ -40,7 +40,7 @@ func TestWeightedOperations(t *testing.T) {
 
 	weightesOps := simulation.WeightedOperations(appParams, cdc, app.AccountKeeper, app.BankKeeper, app.SlashingKeeper, app.StakingKeeper)
 	for i, w := range weightesOps {
-		operationMsg, _, _ := w.Op()(r, app.BaseApp, ctx, accs, ctx.ChainID())
+		operationMsg, _, _, _ := w.Op()(r, app.BaseApp, ctx, accs, ctx.ChainID())
```

Comment:
Is the last return an error that should be handled?

Reply:
Yes, changed!